src/pages/ProviderEServiceCreatePage/components/EServiceCreateStepGeneral/EServiceCreateStepGeneral.tsx

```diff
@@ -25,6 +26,8 @@ export type EServiceCreateStepGeneralFormValues = {
 }
 
 export const EServiceCreateStepGeneral: React.FC = () => {
+  const disabledStage = ['PROD', 'UAT']
+  const isDisabled = disabledStage.includes(STAGE) //check on the environment
```

Comment:
Can you make the constant name more descriptive? 
What is disabled? The whole form? A specific field?